src/test_OilMaster.py

```diff
@@ -0,0 +1,10 @@
+from OilMaster import OilMaster
+
+def test_OilMaster():
+    data_division_file = 'ola_copybook.txt'
+    data_division_file = 'a.txt'
+    data_division_file = 'b.txt'
+    data_division_file = 'o.txt'
+    om = OilMaster(data_division_file)
+
+    assert(om.get_format() is not None)
```

Comment:
again this needs to be fleshed out more
but first priority is getting the EBCDIC files parsed.